ledger/participant-integration-api/src/main/scala/platform/store/DbType.scala

```diff
@@ -48,7 +48,7 @@ private[platform] object DbType {
     case pg if pg.startsWith("jdbc:postgresql:") => Postgres
     case oracle if oracle.startsWith("jdbc:oracle:") => Oracle
     case _ =>
-      sys.error(s"JDBC URL doesn't match any supported databases (h2, pg, oracle)")
+      sys.error(s"JDBC URL '$jdbcUrl' does not match any supported database (h2, postgresql, oracle)")
```

Comment:
this looks like you leak passwords in the error. afaik we deliberately tried to avoid that everywhere at some point so going back to printing that out seems at least worth some consideration.

Reply:
Cmp. https://github.com/digital-asset/daml/pull/8601

Reply:
o.O -- yes, that is indeed a problem. I've added it in the context of a command-line-tool call to `java -Xmx4g -jar bazel-bin/ledger/ledger-on-memory/app_deploy.jar dump-index-metadata` where the error message wasn't helping me understand what was wrong with my string.

Thanks for catching this. Let's close the PR then.

Reply:
fwiw in the JSON API we print jdbc urls in a few places but with passwords stripped which provides a middleground between debuggability & security.